src/windows/editor_object_selection.c

```diff
@@ -903,6 +912,8 @@ static void window_editor_object_selection_resize(rct_window *w)
 	window_set_resize(w, 600, 400, 1200, 1000);
 }
 
+#define FILTER_DROPDOWN_POSITION_SELECTED 5
```

Comment:
Generally we use `DDIX` for dropdown index. so this would be called `DDIX_FILTER_SELECTED`. Also use an enum instead of `#define`. It would be a good idea to add the other dropdown index names as well for consistency.